cloud/terraform/cmd/terraform-machine-controller/Makefile

```diff
@@ -18,7 +18,7 @@ GCR_BUCKET = k8s-cluster-api
 PREFIX = gcr.io/$(GCR_BUCKET)
 DEV_PREFIX ?= gcr.io/$(shell gcloud config get-value project)
 NAME = terraform-machine-controller
-TAG = 0.0.3
+TAG = 0.0.5
```

Comment:
@krousey - I bumped this from 0.0.3 --> 0.0.5 to avoid conflicts w/ your in-flight PR.